topics/userscript/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: userjs, greasemonkey, tampermonkey
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: '2005-03-28'
+short_description: A kind of lightweight browser extension, used to modify webpages
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension
+---
+A userscript is a kind of lightweight and less priveleged browser extension written in ECMAScript used to modify webpages. Userscripts may use some API usually referred as ```GM_API``` allowing them to bypass same-origin policy and store the data in a separate storage. The technology was initially introduced in GreaseMonkey extension for Firefox web browser and then spread to other browsers, sometimes built-in, like in Presto-based Opera, sometimes with help of extensions, like in Chromium-based browsers. The most famous catalogs of userscripts are [GreasyFork](https://greasyfork.org/) and [OpenUserJS](https://openuserjs.org/).
```

Comment:
- JavaScript rather than ECMAScript may be more accessible?
- Just use one backtick around either side of `GM_API`.